tokio/src/sync/broadcast.rs

```diff
@@ -691,6 +691,35 @@ impl<T> Drop for Sender<T> {
 }
 
 impl<T> Receiver<T> {
+    /// Returns the number of messages that were sent into the channel and that
+    /// this [`Receiver`] has yet to receive.
```

Comment:
This should document what happens when the receiver gets so far behind that it will receive a `Lagged` error.